src/test/java/com/akademia/educators_back/mapper/CommentMapperTest.java

```diff
@@ -0,0 +1,83 @@
+package com.akademia.educators_back.mapper;
+
+import com.akademia.educators_back.entity.CommentEntity;
+import com.akademia.educators_back.to.CommentTo;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+public class CommentMapperTest {
+
+    @Autowired
+    private CommentMapper commentMapper;
+
+    @Test
+    void shouldMapToEntityWithProperToValues() {
+        //given
+        CommentTo commentTest = CommentTo.builder()
+                .id(1L)
+                .answer("foo")
```

Comment:
ogólnie jest ok ale warto przy większych testach wyciągać dane do zmiennych i potem używać tylko zmiennej, czyli np:
String answer = "coś tam";
Builder....
   .withAnswer(answer)
   ....
//assert
assertEquals( answer, cośtam.GetAnswer());

daje to tyle że w przypadku zmiany danych mamy tylko jedno miejsce gdzie trzeba to zrobić, tylko jak pisałem sens tego podejścia widać przy bardziej rozbudowanych testach :)